guides/common/modules/proc_configuring-alternate-content-sources.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources* > *Add Source*.
+. Set the *Source type* as *Custom*.
+. Select the *Content type*.
+. Enter a name in the *Name* field and optionally provide a description in the *Description* field.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Enter the Base URL of the alternate content source.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Manual Authentication* or *Content Authentication* credentials, if these are needed.
+. If SSL verification is required, enable *Verify SSl*, and select the SSL CA certificate.
+. Review details and click *Add*.
+. Refresh the alternate content sources by executing the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources list
+# hammer alternate-content-sources refresh --id _alternate content source_ID_
+----
+
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create --name 'example' --base-url http://local-repo.example.com:port --smart-proxy-ids _{SmartProxy}_ID_
```

Comment:
```suggestion
# hammer alternate-content-sources create \
--base-url "_http://local-repo.example.com:port_" \
--name "_My_Alternate_Content_Source_Name_" \
--smart-proxy-ids _{SmartProxy}_ID_
```

Should this example use "https"?

Reply:
@ianballou could you help with this too?

Reply:
It could be HTTP or HTTPS. We can have help text say `https` just to give a nudge towards being more secure.

Reply:
@ianballou @maximiliankolb I'll change the example to say `https`. Thanks :)